tests/test_elections.py

```diff
@@ -105,10 +106,7 @@ def setUp(self):
                 disbursements=75,
                 committee_id=self.committees[0].committee_id,
                 coverage_end_date=datetime.datetime(2012, 9, 30),
-                last_beginning_image_number=123,
```

Comment:
Yes, exactly. These were some fields that we shoehorned into this endpoint when the frontend required them. It doesn't anymore, and the endpoint is clearer and faster without them.